build/build.py

```diff
@@ -67,8 +67,33 @@ def apply_patch(patch):
             print(test["package"], "-run", testName)
             print(go.test(test["package"], "-run", testName))
 
+def parse_source(remote):
```

Comment:
Can we do `remote.lower().strip("/")` to normalize these around case sensitivity and trailing slashes?

I'd also maybe do the final else condition as:

```
return remote.split("/")[-1]
```

So it would still have a potentially unique indicator other than "unknown"